staff/francisco-fernandez/app-template/contact.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Contact</title>
+</head>
+<body>
+    <h3>Contact Form</h3>
```

Comment:
If `<h3>Contact Form</h3>` is the page title would be nice to wrap it with a `<header>` tag. If it is the `<form>` title the should be inside the form tag.

**Form example:**
```
<form action="" method="post">
  <fieldset>
    <legend>Title</legend>
    <input type="radio" name="radio" id="radio">
    <label for="radio">Click me</label>
  </fieldset>
</form>
```

**Further information:**

- [MDN Header Tag](https://developer.mozilla.org/ca/docs/Web/HTML/Element/header)

- [MDN Use of Heading Elements](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements)

Reply:
done, thanks